openstudioreleases/README.md

```diff
@@ -1,6 +1,48 @@
 # OpenStudioCoalition GitHub IO Release Pages
 
+## Install Yarn and package dependencies
+Install the `yarn` package manager. Assuming you have installed nodejs and npm, `npm install -g yarn`
+
+
+Now install the dependencies:
+
+```shell
+yarn install
+```
+
+## Setup secrets
+
+### Download Firebase config file
+
+Go to [Test Project](https://console.firebase.google.com/u/2/project/osc-downloads/overview), you can access this with the OSC email.
+
+https://support.google.com/firebase/answer/7015592?hl=en#zippy=%2Cin-this-article
+
+Go to the Test Project's Settings, get values from the osc "Config" snippet and place it at `firebase-config.json` next to this readme, changing the format to be a JSON one:
+
+```json
+{
+  "apiKey": "XXXXX",
+  "authDomain": "XXXXX",
+  "projectId": "osc-downloads",
+  "storageBucket": "XXXXX",
+  "messagingSenderId": "XXXXX",
+  "appId": "XXXXX",
+  "measurementId": "XXXXX"
+}
+```
+
+Additionally, do `cp .env.example .env`
+
+Then edit it to add the values to each key.
+
+### Google Analytics
+
+Set the `GA_ID` in the `.env` file. This is the Measurement ID for the OS App website itself (OS App Git - GA4)
```

Comment:
@macumber I am not sure at all this is right. https://github.com/openstudiocoalition/openstudiocoalition.github.io/issues/15#issuecomment-2367795599